### PR TITLE
New LNLS Exchange format

### PR DESCRIPTION
**Pull request by ravescovi**

### CODE REVIEW

File: tomopy/io/exchange.py [hunk 1]
```diff
@@ -63,12 +62,11 @@
 logger = logging.getLogger(__name__)
 
 
-__author__ = "Doga Gursoy, Luis Barroso-Luque"
+__author__ = "Doga Gursoy"
 __credits__ = "Francesco De Carlo"
 __copyright__ = "Copyright (c) 2015, UChicago Argonne, LLC."
 __docformat__ = 'restructuredtext en'
 __all__ = ['read_als_832',
-           'read_als_832h5',
```
Comment (tacaswell) [inline tomopy/io/exchange.py]: Why remove this reader?

Comment (ravescovi) [inline tomopy/io/exchange.py]: I think my version was not fully up-to-date. There is also some changes in Petra that i did not want to make.

**Comment by tacaswell** [comment]
I rebased and cleaned up your commit at https://github.com/tomopy/tomopy/compare/master...tacaswell:LNLS?expand=1

I suggest that you add my repo as a remote and then force-push 541a3d8 to your master branch which will update this PR with only the changes you wanted.

**Comment by tacaswell** [comment]
replaced by #165

**Pull request closed without merging**